### DISCUSSION TITLE
upstream failed 是怎么回事啊？

### DISCUSSION THREAD

**Discussion by cresky-github** [IrineSistiana/mosdns#193] [Q&A]
总是有类似

upstream failed	{"query": "ogs.google.com. IN A 51767 1395", "from": "https://doh.opendns.com:443/dns-query", "error": "dns: id mismatch"}

{"query": "play.googleapis.com. IN A 18399 1356", "from": "https://dns.google:443/dns-query", "error": "dns: id mismatch"}

的提示，怎么一回事啊？怎么处理？

**Comment by IrineSistiana** [comment]
按 issue 的模板报告 bug。
